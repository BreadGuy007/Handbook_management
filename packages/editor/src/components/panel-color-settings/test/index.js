/**
 * External dependencies
 */
import { shallow } from 'enzyme';
import { noop } from 'lodash';

/**
 * Internal dependencies
 */
import { PanelColorSettings } from '../';

describe( 'PanelColorSettings', () => {
	it( 'matches the snapshot', () => {
		const wrapper = shallow(
			<PanelColorSettings
				title="Test Title"
				colors={ [] }
				colorSettings={ [
					{
						value: '#000',
						onChange: noop,
						label: 'border color',
					},
					{
						value: '#111',
						onChange: noop,
						label: 'background color',
					},
				] }
			/>
		);

		expect( wrapper ).toMatchSnapshot();
	} );
} );
