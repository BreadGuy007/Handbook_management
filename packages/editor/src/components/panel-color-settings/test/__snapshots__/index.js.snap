// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelColorSettings matches the snapshot 1`] = `
<PanelBody
  className="editor-panel-color-settings"
  title={
    <span
      className="editor-panel-color-settings__panel-title"
    >
      Test Title
      <ColorIndicator
        aria-label="(border color: #000)"
        colorValue="#000"
      />
      <ColorIndicator
        aria-label="(background color: #111)"
        colorValue="#111"
      />
    </span>
  }
>
  <WithColorContext(ColorPaletteControl)
    key="0"
    label="border color"
    onChange={[Function]}
    value="#000"
  />
  <WithColorContext(ColorPaletteControl)
    key="1"
    label="background color"
    onChange={[Function]}
    value="#111"
  />
</PanelBody>
`;
